convert/converter_utils/utils.py

```diff
@@ -182,6 +182,13 @@ def clear():
     os.system(command)
 
 
+def user_confirmed(confirmation):
```

Comment:
@inirudebwoy you gave me the idea for this case while thinking about the one you mentioned. 😄 
Thanks!